lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -65,12 +64,12 @@ export default function AssignmentActivity() {
         <h2 className="text-lg text-brand font-semibold" data-testid="title">
           {assignment.isLoading && 'Loading...'}
           {assignment.error && 'Could not load assignment title'}
-          {assignment.data && title}
+          {assignment.data && assignment.data.title}
         </h2>
       </div>
       <OrderableActivityTable
         loading={students.isLoading}
-        title={assignment.isLoading ? 'Loading...' : title}
+        title={assignment.data?.title ?? 'Loading...'}
```

Comment:
What does this look like before/after if loading fails with an error?

Reply:
Hhm, good point. I think it would stay as "loading..." forever.

Reply:
I pushed another commit fixing this and correcting it for the course section too.